Mueez Afzal-weatherman.py

```diff
@@ -0,0 +1,304 @@
+    def set_lowest(self,month_coldest,day_cold,lowest):
+       self.month_coldest=month_coldest  
+       self.day_cold=day_cold
+       self.lowest=lowest
+   
+    def set_humidity(self,month_humidity,day_humidity,humidity):
+          self.day_humidity=day_humidity
+          self.month_humidity=month_humidity
+          self.humidity=humidity
+    
+    def print_name(self):
+          print("Year:{} \n Max temperature {} C at {} {} \n Min temperature {} C at {} {} \n Humidity {}% at {} {}".format(self.name,self.highest,self.month_hotest,self.day_hot,self.lowest,self.month_coldest,self.day_cold,self.humidity,self.month_humidity,self.day_humidity))
+
+
+    
+        
+  
+class month:
+    def __init__(self):
+          self.year=None
+          self.name=None
+          self.avg_highest=None
+          self.avg_lowest=None
+          self.humidity=None
+
+    def get_year(self):
+         return self.year
+    def get_month(self):
+         return self.name
+    
+    def set_year(self, name):
+          self.year=name
+
+    def set_data(self,name,avg_lowest,avg_highest,humidity):
+          self.name=name
+          self.avg_highest=avg_highest
+          self.avg_lowest=avg_lowest
+          self.humidity=humidity
+    def print_name(self):
+          print("Year:{} Month:{}  \n Avg Max temperature {} C \n Avg Min temperature {} C \n Avg Mean Humidity {}% ".format(self.year,self.name,self.avg_highest,self.avg_lowest,self.humidity))
+
+class day:
+    def __init__(self):
+          self.year=None
+          self.month=None
+          self.day=None
+          self.highest=None
+          self.lowest=None
+
+    def get_year(self):
+         return self.year
+
+    def get_month(self):
+         return self.month
+ 
+    
+    def set_year(self, name):
+          self.year=name
+
+    def set_hot(self,hot):
+        self.highest=hot
+
+    def set_cold(self,cold):
+        self.lowest=cold
+
+    def set_data(self,year,month,day):
+          self.year=year
+          self.month=month
+          self.day=day
+
+    def print_name(self):
+          print("\n Year :{} Month :{} Day :{} ".format(self.year,self.month,self.day))
+          print("Highest temperature:",end="")
+          if self.highest==None:
+              print("NO Record")
+          elif self.highest<=0:
+              for i in range(self.highest,0):
+                  print ("+",end="")
+          else:
+              for i in range(self.highest):
+                  print ("+",end="")
+          print(" ",self.highest,"C")
+          print("Coldest temperature:",end="")
+          if self.lowest==None:
+              print("NO Record")
+          elif self.lowest<=0:
+              for i in range(self.lowest,0):
+                 print ("-",end="")
+          else:
+              for i in range(self.lowest):
+                 print ("-",end="")
+          print(" ",self.lowest,"C")
+
+def file_data(day_list,month_list,year_list):
+    day_count,month_count,year_count=0,0,0

```

Comment:
this information is passed by the user through the commandline args. you don't need to calculate it on run time.